### ISSUE TITLE
Nothing happens when running the flameshot gui command

### ISSUE THREAD

**Issue by arnav7633** [flameshot-org/flameshot#2127]
### Flameshot Version

v0.10.2

### Installation Type

Compiled from source

### Operating System type and version

Arch 64bit, Custom compiled linux tkg with pds

### Description

Hey! Whenever I try running the `flameshot gui` command nothing happens. Nothing happens, not even the "Unable to capture" as I am using gnome 41/mutter. I was reviewing some of the issues and while running the flameshot daemon after killing it in the terminal and trying to take capture the daemon logs nothing
```
 pkill flameshot
                             flameshot

Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
```

### Steps to reproduce

1. Kill flameshot
2. Run flameshot daemon
3. Run `flameshot gui`
4. The daemon remains empty and nothing happens

### System Information

System:
  Host: arch Kernel: 5.15.4-221-tkg-pds x86_64 bits: 64
  Desktop: GNOME 41.1 Distro: Arch Linux
Graphics:
  Device-1: AMD Mullins [Radeon R4/R5 Graphics] driver: amdgpu v: kernel
  Device-2: AMD Sun XT [Radeon HD 8670A/8670M/8690M / R5 M330 / M430 / Radeon
  520 Mobile]
  driver: amdgpu v: kernel
  Device-3: IMC Networks EasyCamera type: USB driver: uvcvideo
  Display: wayland server: X.Org 1.21.1.3 driver: loaded: amdgpu
  note: n/a (using device driver) - try sudo/root resolution: 1366x768~60Hz

**Comment by Araly** [comment]
I have this same issue.

If I run `flameshot` I get:
```

**Comment by borgmanJeremy** [comment]
Make sure you install xdg-desktop-portal and xdg-desktop-portal-gnome

Then logout and back in. Please let me know if that fixes it and I will get this merged into the documentation / arch wiki.

**Comment by Araly** [comment]
> Make sure you install xdg-desktop-portal and xdg-desktop-portal-gnome
> 
> Then logout and back in. Please let me know if that fixes it and I will get this merged into the documentation / arch wiki.

It works, though now it also requires me to click the share screenshot button of gnome. I imagine this is a wayland thing, improving security and the like so I won't complain, but I have to admit it feels less smooth than before.

Anyways, good job, it works now ^^

**Comment by mmahmoudian** [comment]
@Araly nope, that is a Gnome 41 Wayland thing thanks to a decision their dev team made. Checkout the pinned issues in our repo for detailed information, and please take your complaints to Gnome devs (you can find the links in #1910 and #2186). If you don't want to see the "share" message, simply use KDE Wayland, or Gnome Xorg if you have to stick to Gnome. Literally any Wayland DE or Wayland window manager other than Gnome would do!

**Comment by krishna-ezlo** [comment]
I'm getting following issue while running flameshot from commandline:
`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
`

**Comment by mmahmoudian** [comment]
@krishna-ezlo 

> I'm getting following issue while running flameshot from commandline:
> `Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
> `

How do you run Flameshot? Which version? How have you installed it? What is your DE? And apart from that warning, does Flameshot work?

**Comment by krishna-ezlo** [comment]
@mmahmoudian 
The package was broken while installing for the first time.
Latter, I uninstalled and reinstalled it. It's working now. 
Thanks for the response though.

**Comment by 3xpert** [comment]
> @krishna-ezlo
> 
> > I'm getting following issue while running flameshot from commandline:
> > `Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway. `
> 
> How do you run Flameshot? Which version? How have you installed it? What is your DE? And apart from that warning, does Flameshot work?

I get the same issue. 
If I run Flameshot from icon-right click-flameshot, nothing will happen.
If I run command flameshot gui from commandline, I have to pkill flameshot first, flameshot works. And the second time if I run the same command without pkill flameshot first, flameshot doesn't work. Both get the same warning as above.
manjaro
Flameshot v12.1.0
DE: GNOME 43.1

**Comment by krishna-ezlo** [comment]
> > @krishna-ezlo
> > > I'm getting following issue while running flameshot from commandline:
> > > `Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway. `
> > 
> > 
> > How do you run Flameshot? Which version? How have you installed it? What is your DE? And apart from that warning, does Flameshot work?
> 
> I get the same issue. If I run Flameshot from icon-right click-take screenshot, nothing will happen. If I run command flameshot gui from commandline, I have to pkill flameshot first, flameshot works. And the second time if I run the same command without pkill flameshot first, flameshot doesn't work. Both get the same warning message as above. manjaro Flameshot v12.1.0 DE: GNOME 43.1

I uninstalled it and reinstalled several time
now its working 
It seems flameshot is working differently now. I've to click "share" button to edit the screenshot.

**Comment by 3xpert** [comment]
> > > @krishna-ezlo
> > > > I'm getting following issue while running flameshot from commandline:
> > > > `Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway. `
> > > 
> > > 
> > > How do you run Flameshot? Which version? How have you installed it? What is your DE? And apart from that warning, does Flameshot work?
> > 
> > 
> > I get the same issue. If I run Flameshot from icon-right click-take screenshot, nothing will happen. If I run command flameshot gui from commandline, I have to pkill flameshot first, flameshot works. And the second time if I run the same command without pkill flameshot first, flameshot doesn't work. Both get the same warning message as above. manjaro Flameshot v12.1.0 DE: GNOME 43.1
> 
> I uninstalled it and reinstalled several time now its working It seems flameshot is working differently now. I've to click "share" button to edit the screenshot.

Flameshot worked fine before I upgrade Gnome from 42. And yes, I've to click "Share" button too.  I do have uninstalled and reinstall, but it still does not work.

**Comment by 3xpert** [comment]
Yes, same here. But should I back to Xorg for the reason?

**Comment by muhammadsiddiqbe** [comment]
Is there a way to get it to work on Wayland also? The same error here.

Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
